src/internal/modules/charge-information/pre-handlers.js

```diff
@@ -85,6 +85,24 @@ const loadBillingAccounts = async request => {
   }
 };
 
+const loadLicencesWithoutChargeVersions = async request => {
+  try {
+    const response = await services.water.chargeVersionWorkflows.getLicencesWithoutChargeInformation();
+    return response.data.filter(row => row.licence.expiredDate !== null && moment(row.licence.expiredDate).isAfter(new Date())).sort((rowA, rowB) => new Date(rowA.licence.startDate) - new Date(rowB.licence.startDate));
```

Comment:
I'd prefer if this logic was split out over a few lines for readability - perhaps the filter and sort functions could be defined separately